features/specs/S-en-T-test-omgekeerd-ouders/Tg017.feature

```diff
@@ -165,18 +165,22 @@ Functionaliteit: Tg017 - Rooyakkers-Ros - Getrouwd met kind in Cat.11 1D
       | ingangsdatum geldigheid (85.10)      | 20200301                    |
 
 
-  Scenario: Lg01_097 - gehuwd, 1 minderjarig kind
+  Scenario: Lg01_097 - gehuwd, 1 minderjarig kind, gerechtelijke uitspraak 1D
     # Meerderjarig
-    # Route: 2m - Wie heeft gezag?: meerderjarig, gezag is niet van toepassing (NVT)
     Als gezag wordt gezocht met de volgende parameters
       | naam                | waarde    |
       | burgerservicenummer | 000000012 |
     Dan heeft de response een persoon met de volgende gegevens
       | naam                | waarde    |
       | burgerservicenummer | 000000012 |
-    En heeft de persoon geen gezag
+    En heeft de persoon een 'gezag' met de volgende gegevens
```

Comment:
hier zelfde probleem als eerder genoemd. voor persoon mag alleen gezagsrelatie geleverd worden waar de gevraagde persoon een rol in heeft. dus minderjarige, ouder of derde in is.